src/Perlang.Interpreter/Extensions/TypeExtensions.cs

```diff
@@ -9,6 +9,12 @@ namespace Perlang.Interpreter.Extensions
     internal static class TypeExtensions
     {
         public static string ToTypeKeyword(this Type type)
+        {
+            return ToTypeKeywordNew(type);
+        }
+
+        // TODO: replace usage of the above with this one
+        public static string ToTypeKeywordNew(Type type)
```

Comment:
This mess is something we should clean up, preferably within the scope of this MR. We could also look into merging it with `Utils.StringifyType()` which has some semi-related logic as well.

In the end, we should have **one** stastic method for this and it should have perhaps two overloads:

- One that takes an `object obj` parameter, which can be nullable.
- One that takes a `Type type` parameter. (Will this even be needed? Time will tell.)

Reply:
Fixed in #228, resolving.